config/environments/public/public.prover.config.json

```diff
@@ -1,6 +1,6 @@
 {
     "runProverServer": false,
-    "runProverServerMock": true,
+    "runProverServerMock": false,
```

Comment:
yep, but not locally, we don't need to run a prover locally because we don't run an aggregator, it's configured properly in the public testnet infrastructure

we just need this `executor` running in order to process txs locally to sync the testnet state and to estimate txs gas